frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -523,11 +527,85 @@ function ShouldUseTreesFormItem() {
   );
 }
 
```

Comment:
Please add a type for the settings created / configured by this component here in the lines above the component. This can be used to improve the typing of the data received from the form component. the correct type should be something like:

```suggestion
type SplitMergerEvaluationSettings = {
    useSparseTracing?: boolean;
    maxEdgeLength?: number;
    sparseTubeThresholdInNm?: number;
    minimumMergerPathLengthInNm?: number;
}
```
The settings are numbers because you are correctly using number inputs and not simple string inputs.